ui/apps/platform/src/Containers/SystemConfig/Form/SystemConfigForm.tsx

```diff
@@ -531,6 +532,23 @@ const SystemConfigForm = ({
                             </FormGroup>
                         </GridItem>
                     </Grid>
+                    <Title headingLevel="h2">Prometheus metrics configuration</Title>
+                    <Grid hasGutter>
+                        <PrometheusMetricsForm
+                            pcfg={values?.privateConfig}
+                            category="imageVulnerabilities"
+                            title="Image vulnerabilities"
+                            onChange={onChange}
+                            onCustomChange={onCustomChange}
+                        />
+                        <PrometheusMetricsForm
```

Comment:
Currently, there are only two forms. There will come at least two more:

* Node vulnerabilities
* System health

It might make sense to rethink how the forms play together. May be Tabs?

Reply:
Yeah, debt that we have not separated into tabs. Ditto for Integrations page.

I will bring up in UI/UX weekly. If only we can limit the scope to what is doable in 4.10A sprint.

Reply:
Thanks. Let's concentrate on the plain old form cards, then! I've integrated policy violations, to have more than one.